backend/src/main/java/com/redhat/cloud/notifications/auth/rbac/RbacGroup.java

```diff
@@ -0,0 +1,42 @@
+package com.redhat.cloud.notifications.auth.rbac;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+import org.jboss.resteasy.reactive.ClientWebApplicationException;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.ws.rs.core.Response;
+import java.time.Duration;
+import java.util.UUID;
+
+@ApplicationScoped
+public class RbacGroup {
+
+    @Inject
+    @RestClient
+    RbacServer rbacServer;
+
+    @ConfigProperty(name = "rbac.enabled", defaultValue = "true")
+    Boolean isRbacEnabled;
+
+    public boolean validate(UUID groupId, String rhIdentity) {
+        if (isRbacEnabled) {
+            try {
+                Response response = rbacServer.getGroup(groupId, rhIdentity)
+                        .await()
+                        .atMost(Duration.ofSeconds(2L));
+                return response.getStatus() == 200;
+            } catch (ClientWebApplicationException errorException) {
+                if (errorException.getResponse().getStatus() == 404) {
+                    // This is fine, the group was not found
```

Comment:
Or we could have an RBAC host issue in the configuration 😄

Reply:
ummh - it returns a 404 if we have host issues? :D 
Would you like to add other validation here about the content?

Reply:
No it's OK 😃 If the RBAC config was wrong, we would know it before reaching this point anyway.